components/dashboard/src/components/PrebuildLogs.tsx

```diff
@@ -15,8 +15,7 @@ export default function PrebuildLogs(props: { workspaceId?: string }) {
   const [ workspace, setWorkspace ] = useState<Workspace | undefined>();
   const [ workspaceInstance, setWorkspaceInstance ] = useState<WorkspaceInstance | undefined>();
   const [ error, setError ] = useState<Error | undefined>();
-  const logsEmitter = new EventEmitter();
-  const service = getGitpodService();
+  const [ logsEmitter ] = useState(new EventEmitter());
```

Comment:
@jankeromnes logs were sent to the wrong emitter. `const` values should not be state. now I can see logs and log output for prebuilds 👍🏻

Reply:
Aha, exciting!